config/manifests/bases/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -124,7 +124,7 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.48.1
+  replaces: jaeger-operator.v1.51.0
```

Comment:
Shouldnt that be 1.49.0?

Reply:
Does this block upgrades? cc @pavolloffay @IshwarKanse

Reply:
OLM uses the replaces: field for upgrade path, we should fix it.